WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableView.swift

```diff
@@ -0,0 +1,123 @@
+import UIKit
+import WordPressShared
+
+// wrap BlogListDataSource calls to add a section for the user's info cell
+class LoginEpilogueTableView: UITableViewController {
```

Comment:
@nheagy sir, this is a UITableViewController, but the class name is `TableView`. A bit confusing!